docs/source/developer-guides/prs-and-reviews.rst

```diff
@@ -0,0 +1,75 @@
+Consolidation:
+==============
+
+Overarching Goals of good Review
+--------------------------------
+
+The goal of a review is to ensure that the proposed change is useful and maintainable over the long run.
+
+PR Description
+---------------
+
+- Have a meaningful title: it's easier when creating the changelog or when searching through old PR
+- Motivate why we should merge the PR: adding/changing code risks introducing a new bug. IMO one person asking for a nice to have feature doesn't qualify.
+- Explain the PR does in the description: it makes it easier/faster to review
+- Make the PR as small as possible: it's much easier to review five 200 lines PR than one 1000 lines PR
+- If a PR is known to create conflict with other active PRs, try to coordinate, and link to each other
+  - if appropriate explain desired review/merge order 
+- For complex changes that will require multiple PR, create an issue to keep track of what has been done and what's left to do.
+- Is the PR focused on a single bugfix or feature?
+  - consider moving self contained changes to separate PRs 
+
+Public Interfaces
+-----------------
+
+- are there comprehensive tests
+- does the interface meet the use case - and for that matter is there a use case description
+  - is the usage intuitive
+  - do we really want this as a public interface, and maintain it?
+- is the interface API consistent with existing ones?
+  - e.g. if everything else takes execution space instances as the first argument, don't make it the last argument
+  - does the naming style match other Kokkos APIs
```

Comment:
- Are there guidelines / expectations for naming style?

Reply:
not written down right now.

Reply:
Here's your opportunity ; P

Reply:
But not part of this thing.